tower-balance/src/load/pending_requests.rs

```diff
@@ -8,11 +8,7 @@ use super::{Instrument, InstrumentFuture, NoInstrument};
 
 /// Expresses load based on the number of currently-pending requests.
 #[derive(Debug)]
-pub struct PendingRequests<S, I = NoInstrument>
-where
-    S: Service,
-    I: Instrument<Handle, S::Response>,
-{
+pub struct PendingRequests<S, I = NoInstrument> {
```

Comment:
Struct bounds removed to avoid needing `PhantomData` for request type parameter.